wcomponents-theme/src/test/spec/wc.ui.redirect.test.mjs

```diff
@@ -0,0 +1,81 @@
+import redirect from "wc/ui/redirect.mjs";
+
+describe("wc/ui/redirect", () => {
+	// window = Object.create(Window);
+
+	const delay = 50;  // milliseconds to wait for events and stuff to be actioned
+	const origWindow = window.parent;
+	const replaceSpy = jasmine.createSpy();
+
+	beforeEach(() => {
+		window.parent = {
+			location : {
```

Comment:
no space before `:` except for the ternary operator

For example this line should be:

`location: {`